src/uniprotkb/components/protein-data-views/FreeTextView.tsx

```diff
@@ -93,6 +93,15 @@ export const RichText = ({ children, addPeriod, noLink }: RichTextProps) => (
             </Fragment>
           );
         }
+        const dbSnpMatch = part.match(reDbSnpCapture);
+        if (dbSnpMatch?.groups) {
+          const { rsid } = dbSnpMatch.groups;
+          return (
+            <ExternalLink url={externalUrls.dbSNP(rsid)} key={rsid}>
```

Comment:
`ExternalLink` used here however pubmed is using just a link. Not sure if there is an intention for pubmed to exceptionally not use `ExternalLink`.

Reply:
Disregard, I forgot that pubmed links internally.